src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupRepo.java

```diff
@@ -340,4 +347,62 @@
 
         return retVal;
     }
+
+    @Override
+    public RepoGenericResponse<SuspendedScalingPolicy> suspendScalingPolicies(String groupId, String policyName,
```

Comment:
Change 'groupId' parameter to to 'elastigroupId' for consistency.

Reply:
Incorporated